actor.c

```diff
@@ -90,6 +91,17 @@ const struct actor_type_t actor_types[NBR_OF_ACTOR_TYPES] = {
 		NULL,
 		NULL,
 		NULL
+	},
+	{
+		"io.Button",
+		actor_button_init,
+	    NULL,
```

Comment:
Is init not called if it was a shadow actor? As it is now it seems to be working.